#include "fileSystemManagerImpl.h"

FileSystemManagerImpl::FileSystemManagerImpl() {}
FileSystemManagerImpl::~FileSystemManagerImpl() {}

grpc::Status FileSystemManagerImpl::ListAccountFiles(grpc::ServerContext *context, const file_system::account::ListAccountFilesRequest *request, file_system::account::ListAccountFilesResponse *response)
{

}
grpc::Status FileSystemManagerImpl::GetAccountIntroFile(grpc::ServerContext *context, const file_system::account::GetAccountIntroFileRequest *request, file_system::account::GetAccountIntroFileResponse *response)
{

}
grpc::Status FileSystemManagerImpl::GetAccountTermFile(grpc::ServerContext *context, const file_system::account::GetAccountTermFileRequest *request, file_system::account::GetAccountTermFileResponse *response)
{
    
}